### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by jeiholzer**
Please review.

### CODE REVIEW

File: test/components/main.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ import TestHarnessComponent from './test-harness-component.js';
 
 const module = angular.module('bedrock.agreement-test', [
   'bedrock.agreement', 'bedrock.authn', 'bedrock.authn-password',
-  'bedrock.resolver', 'bedrock.session', 'ngMaterial'
+  'bedrock.resolver', 'bedrock.session', 'ngMaterial', 'ngMessages',
+  'bedrock.form'
```
Comment (mattcollier) [inline test/components/main.js]: We have a convention to keep these alphabetized.

File: test/components/test-harness-component.html [hunk 2]
```diff
@@ -7,13 +7,48 @@
...
+      theme: 'material',
+      type: 'email',
+      showValidation: true
+    }" required>
     <br-input-help>
-      Enter a unique short name for this identity.
+      Enter a unique email for this identity.
     </br-input-help>
+    <br-input-validation-errors>
+      <p ng-show="$ctrl.identityForm.slug.$error.required">
```
Comment (mattcollier) [inline test/components/test-harness-component.html]: This should be utilizing the newer ng-messages method as shown in angular-authn-password.

File: test/protractor/pages/app.js [hunk 3]
```diff
@@ -7,7 +7,8 @@ module.exports = api;
 api.login = function(identity) {
   element(by.buttonText('Sign In')).click();
   const c = $('br-authn-password');
-  c.element(by.brModel('$ctrl.sysIdentifier')).sendKeys(identity.sysIdentifier);
+  c.element(by.brModel('$ctrl.sysIdentifier'))
+    .sendKeys(`${identity.sysIdentifier}`);
```
Comment (mattcollier) [inline test/protractor/pages/app.js]: should be no need for wrapping this in backticks, so just should suffice.
```
.sendKeys(identity.sysIdentifier);
```

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: test/protractor/pages/app.js (hunk 3 above)
Comment (jeiholzer) [reply test/protractor/pages/app.js]: wondered about that, ok

Comment (mattcollier) [reply test/protractor/pages/app.js]: @jeiholzer Here's a spot where using a template would be useful: https://github.com/digitalbazaar/bedrock-angular-agreement/pull/10/files#diff-671e631aeda5fd27949e7ff95c8c4c11R16

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: test/components/test-harness-component.html (hunk 2 above)
Comment (jeiholzer) [reply test/components/test-harness-component.html]: Matt, I made these changes but now I am not seeing the error messages. I compared the view code between agreement and autn password and do not seem to be missing anything. I believe I have all the necessary

File: test/components/test-harness-component.html [hunk 4]
```diff
@@ -3,17 +3,52 @@
...
+<form name="$ctrl.identityForm" class="br-flex-vertical-children br-flex-stretch-children-width">
   <br-input br-model="$ctrl.sysSlug"
     br-options="{
       name: 'slug',
-      label: 'Short Name'
-    }">
+      label: 'Email Address',
+      theme: 'material',
+      type: 'email',
+      showValidation: true
```
Comment (mattcollier) [inline test/components/test-harness-component.html]: in response to your difficulty with errors.  I believe with the material theme, showValidation should be removed.  I noted that that seems to be the only difference between implementation here and in bedrock-angular-authn-password.

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: test/components/test-harness-component.html (hunk 2 above)
Comment (mattcollier) [reply test/components/test-harness-component.html]: See my comment on test-harness-component.html.

Comment (jeiholzer) [reply test/components/test-harness-component.html]: Turns out I was not formatting the 'html' correctly

**Comment by jeiholzer** [comment]
I think that I have address everything, please review again

**Review by mattcollier — APPROVED** [review]

**Comment by mattcollier** [comment]
@jeiholzer nicely done.  Please squash onto 6d1c6dc and fix the typo in the commit message there.

**Pull request merged**